### PR TITLE
Change HID bubble title to Brave Wallet for page and panel

### PR DESCRIPTION
**Pull request by spylogsster**
Resolves https://github.com/brave/brave-browser/issues/19130

- Changed title of the hid permission bubble to Brave Wallet for wallet panel and page

image

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:

- For wallet page is enough to import accounts from unknown device (or with clean profile)
- For panel need to import accounts from devices, reset hid option and try to sign transaction or message with device, the browser will show the dialog from wallet panel in this case

### CODE REVIEW

File: chromium_src/chrome/browser/ui/views/permission_bubble/chooser_bubble_ui.cc [hunk 1]
```diff
@@ -9,17 +9,34 @@
...
+                        GURL(kBraveUIWalletPanelURL));
+}
+
+bool IsBraveWalletPage(content::WebContents* content) {
+  return content && content->GetVisibleURL().EqualsIgnoringRef(
+                        GURL(kBraveUIWalletPageURL));
+}
+
+std::unique_ptr<views::Label> BraveCreateTitleOriginLabel(
+    const std::u16string& source_text) {
```
Comment (goodov) [inline chromium_src/chrome/browser/ui/views/permission_bubble/chooser_bubble_ui.cc:35]: `source_text` is not used.

Comment (spylogsster) [reply chromium_src/chrome/browser/ui/views/permission_bubble/chooser_bubble_ui.cc]: it was actually unused but I removed that patches

**Review by bbondy — APPROVED** [review]

File: chromium_src/chrome/browser/ui/hid/hid_chooser_controller.cc [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+#include "chrome/grit/generated_resources.h"
+#include "components/grit/brave_components_strings.h"
+#include "content/public/browser/web_contents.h"
+#include "ui/base/l10n/l10n_util.h"
+#include "url/gurl.h"
+#include "url/origin.h"
+
+namespace {
+
+bool IsBraveWalletOrigin(const url::Origin& origin) {
```
Comment (bridiver) [inline chromium_src/chrome/browser/ui/hid/hid_chooser_controller.cc:17]: this seems like it should be a utility method in brave wallet

Comment (spylogsster) [reply chromium_src/chrome/browser/ui/hid/hid_chooser_controller.cc]: done

**Review by bridiver — APPROVED** [review]

**Pull request merged**